keras/layers/reshaping/up_sampling2d.py

```diff
@@ -65,7 +65,8 @@ class UpSampling2D(Layer):
       It defaults to the `image_data_format` value found in your
       Keras config file at `~/.keras/keras.json`.
       If you never set it, then it will be "channels_last".
-    interpolation: A string, one of `nearest` or `bilinear`.
+    interpolation: A string, one of `area`, `bicubic`, `bilinear`, `gaussian`,
```

Comment:
Likewise

Reply:
So should this one be `"area"` or "area" ?

Reply:
@fchollet

Reply:
Changed in [Fix string formatting](https://github.com/keras-team/keras/pull/16249/commits/aa8af10a160845df30a8b28470e866e02c68bdf5)